airflow/hooks/hive_hooks.py

```diff
@@ -337,6 +338,26 @@ def check_for_partition(self, schema, table, partition):
         else:
             return False
 
+    def check_for_named_partition(self, schema, table, partition_name):
```

Comment:
Nit: docstring (I know the other methods didn't have one but we should add them going forward)